programs/serum/CancelOrder.go

```diff
@@ -0,0 +1,225 @@
+//
```

Comment:
I was thinking that one way to test this would be to use this code in the sample code in serum-api for sending transactions?

You could send a transaction, cancel the order, and then make sure that the order is not in the orderbook?

Reply:
Yup, working on that atm :-)